astroML/linear_model/linear_regression_errors.py

```diff
@@ -51,8 +51,8 @@ def fit(self, X, y, y_error=1, x_error=None, *,
             eta = pm.Normal('eta', mu=(tt.dot(slope.T, ksi.T) + inter),
                             sigma=int_std, shape=y.shape)
 
-            # observed xi, yi
-            x = pm.Normal('xi', mu=ksi.T, sigma=x_error, observed=X, shape=X.shape)
+            # observed xi, yi - we don't need xi here though.
```

Comment:
I don't yet see why, let's keep this separate from this PR, maybe in an issue as it could be source of a bug I see.

Reply:
Left a low-priority reminder issue for this: https://github.com/astroML/astroML/issues/233#issue-815769018